### DISCUSSION TITLE
which tree should be selected to display

### DISCUSSION THREAD

**Discussion by iaunicorn** [hahnlab/CAFE5#110] [Q&A] [answered]
Hi,

I am confusing how to display the results/Base_asr.tre.
It was:
BEGIN TREES;
  TREE 0 = ((((cat<11>_59:68.7105,horse<10>_62:68.7105)<15>_62:4.56678,cow<14>_63:73.2773)<19>_62:20.7227,(((((chimp<1>_61:4.44417,human<0>_66:4.44417)<3>_63:6.68268,orang<2>_61:11.1268)<7>_63:2.28586,gibbon<6>_63:13.4127)<9>_63:7.21153,(macaque<5>_64:4.56724,baboon<4>_62:4.56724)<8>_63:16.057)<13>_63:16.0607,marmoset<12>_66:36.6849)<18>_64:57.3151)<21>_62:38.738,(rat<17>_63:36.3024,mouse<16>_61:36.3024)<20>_62:96.4356)<22>_62;
  TREE 1 = ((((cat<11>_57:68.7105,horse<10>_55:68.7105)<15>_55:4.56678,cow<14>_54:73.2773)<19>_55:20.7227,(((((chimp<1>_55:4.44417,human<0>_52:4.44417)<3>_54:6.68268,orang<2>_54:11.1268)<7>_54:2.28586,gibbon<6>_54:13.4127)<9>_54:7.21153,(macaque<5>_56:4.56724,baboon<4>_57:4.56724)<8>_56:16.057)<13>_55:16.0607,marmoset<12>_56:36.6849)<18>_55:57.3151)<21>_55:38.738,(rat<17>_53:36.3024,mouse<16>_52:36.3024)<20>_53:96.4356)<22>_54;
  TREE 2 = ((((cat<11>*_37:68.7105,horse<10>_41:68.7105)<15>_44:4.56678,cow<14>_49:73.2773)<19>_44:20.7227,(((((chimp<1>_51:4.44417,human<0>*_76:4.44417)<3>*_52:6.68268,orang<2>*_41:11.1268)<7>_44:2.28586,gibbon<6>_45:13.4127)<9>_44:7.21153,(macaque<5>*_36:4.56724,baboon<4>_39:4.56724)<8>*_39:16.057)<13>_42:16.0607,marmoset<12>_37:36.6849)<18>_41:57.3151)<21>*_45:38.738,(rat<17>_67:36.3024,mouse<16>*_79:36.3024)<20>*_69:96.4356)<22>_51;
.......
There are more than 3000 trees. I am wondering which one can be selected to display.
I really appreciate any help you can provide.

**Comment by hahnlab-user** [comment, marked as answer]
Hi,

Each tree in this file plots the results for a different family. So you just pick whichever one you want to view/analyze.


Matt

**Comment by iaunicorn** [comment]
Hi Matt,

Thanks so much for your reply. Another question is how to count the number of "Rapidly evolving families". I found CAFE5 didn't provide this information.

Best wishes,
Alejandra

**Comment by iaunicorn** [comment]
Hi Matt,

If I understand correctly, I can directly select Rapidly evolving families based on asterisk marks in Base_asr.tre file, right?
For example, I count the number of rapidly evolving families in cat:
grep "cat<11>\*" Base_asr.tre |awk '{print $2}' |wc -l

Alejandra

**Comment by hahnlab-user** [comment]
Yes, that should be right!
